pegasus/sites.v3/hourofcode.com/i18n/en.yml

```diff
@@ -10498,6 +10498,68 @@
+        button: "Explore AI curriculum"
+      pl:
+        title: "Professional learning"
+        desc: "Get a hands-on introduction to the artificial intelligence and machine learning curriculum. Train a machine learning model using AI Lab, and engage with machine learning apps."
+        button: "See professional learning"
+      videos:
+        title: "AI videos"
+        desc: "Videos that introduce you to how artificial intelligence works and why it matters. Then delve into issues like algorithmic bias and the ethics of AI decision-making."
+        button: "Watch AI videos"
+    form:
+      heading: "Stay informed on AI at Code.org"
+      desc: "Keep up with Code.org's innovative AI tools and updates. Subscribe to our newsletter for the latest insights and developments."
+    faq:
+      heading: "Frequently asked questions"
+      support:
+        question: "What can the AI Teaching Assistant currently support?"
+        answer_01: "The AI Teaching Assistant's first feature is AI-supported rubrics in the Interactive Games and Animations unit of Code.org's Computer Science Discoveries (CSD) curriculum. This feature:"
+        list_01: "Allows teachers to assess their students' coding projects within the Code.org platform"
+        list_02: "Saves teachers time by providing an initial assessment of student work"
+        list_03: "Helps build confidence for teachers who are new to teaching Computer Science"
+        answer_02: "In the future, the AI Teaching Assistant's feature set will expand to assist teachers with additional tasks."
+      units:
+        question: "What Units is the AI Teaching Assistant available for?"
+        answer: "The AI Teaching Assistant is launching in CSD's Interactive Games and Animations Unit. The team is hard at work building AI-supported features for additional curriculum."
+      available:
+        question: "When will the AI Teaching Assistant be available to all Interactive Games and Animation users?"
+        answer: "The AI Teaching Assistant will launch to all CSD teachers in August 2024, in time for the 2024-2025 school year."
+      access:
+        question: "How do I access the AI Teaching Assistant?"
+        answer: "The AI Teaching Assistant is available for teachers accepted into the pilot in mini and end-of-unit projects in the Interactive Games and Animations unit. In those lessons, the AI Teaching Assistant will appear in the bottom left-hand corner of your screen."
+      keep_up:
+        question: "Where should I go to keep up with Code.org's investment in AI-supported tools?"
+        answer: "Code.org is investing in using AI to enhance teaching and learning.  Check out [https://code.org/ai](https://code.org/ai) to learn more about our AI curriculum, professional learning, and tools."
```

Comment:
Nit: should this be "[code.org/ai](https://code.org/ai)"? I feel like elsewhere on the site we omit the `https://` in the text shown

Reply:
Good catch!